src/seedu/addressbook/data/AddressBook.java

```diff
@@ -54,6 +54,13 @@ public void removePerson(ReadOnlyPerson toRemove) throws PersonNotFoundException
         allPersons.remove(toRemove);
     }
 
+    /**
+     * Sorts all person in the address book by their name in alphabetical order.
+     */
+    public void sortPerson(){
```

Comment:
> Java reserved words should be followed by a white space.

Refer to [CS2113T Java Coding Standard](https://nuscs2113-ay1819s2.github.io/website/coding-standards/java/intermediate.html) for more information.